src/room_member.cpp

```diff
@@ -137,6 +161,17 @@ std::deque<WifiPacket> RoomMember::PopWifiPackets(WifiPacket::PacketType type, c
     }
 }
 
+void RoomMember::SendChatMessage(const std::string& message) {
+    RakNet::BitStream stream;
+
+    RakNet::RakString nickname = this->nickname.c_str();
+    stream.Write(nickname);
```

Comment:
while this works in theory, ideally this would be checked / added by the server.
Keeps bandwidth slightly lower + it will be harder to spoof to be another player.

Theoretically this makes it insanely easy to create a fake client to spam shit in channels with a random name.